code/widgets/org.eclipse.scout.jswidgets.ui.html/src/main/js/tree/TreePropertiesBoxModel.ts

```diff
@@ -7,22 +7,19 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  */
-import {CheckBoxField, GroupBox, GroupBoxModel, SmartField, TreeCheckableStyle} from '@eclipse-scout/core';
-import {CheckableTreeStyleLookupCall} from '../index';
+import {AutoCheckStyle, CheckBoxField, GroupBox, GroupBoxModel, SmartField, TreeCheckableStyle} from '@eclipse-scout/core';
+import {AutoCheckStyleLookupCall, CheckableTreeStyleLookupCall} from '../index';
 
 export default (): GroupBoxModel => ({
   id: 'jswidgets.TreePropertiesBox',
   objectType: GroupBox,
   label: 'Tree Properties',
   fields: [
     {
-      id: 'AutoCheckChildrenField',
-      objectType: CheckBoxField,
-      label: 'Auto Check Children',
-      labelVisible: false,
-      gridDataHints: {
-        fillHorizontal: false
-      }
+      id: 'AutoCheckStyleField',
+      objectType: SmartField,
+      label: 'Auto Check Style',
+      lookupCall: AutoCheckStyleLookupCall
     },
     {
```

Comment:
Maybe add a string field that contains the ids of the checked nodes? This way we could easier verify that the state is correct